app/src/main/java/com/example/parkingsystem/mvp/presenter/ParkingReservationPresenter.kt

```diff
@@ -44,6 +44,26 @@ class ParkingReservationPresenter(
     }
 
     override fun onButtonParkingReservationSavePressed() {
-        view.showOkReservationToast()
+        model.completeReservationInfo(
+            (if (view.getParkingLot().isNotEmpty()) view.getParkingLot().toInt() else Constants.MINUS_ONE),
+            (if (view.getSecurityCode().isNotEmpty()) view.getSecurityCode().toInt() else Constants.MINUS_ONE),
```

Comment:
I think this else block is unnessesary. you have the default parameter

Reply:
As we talked, I will have in mind your advice for the future. Thanks a lot =)